src/tests/classic/run.rs

```diff
@@ -256,3 +264,90 @@ fn test_forms_of_destructuring_allowed_by_classic_1() {
         "(i 2 (q . 2) (q . 3))"
     );
 }
+
+#[test]
+fn test_num_encoding_just_less_than_5_bytes() {
+    let res = do_basic_run(&vec!["run".to_string(), "4281419728".to_string()])
+        .trim()
+        .to_string();
+    assert_eq!(res, "0x00ff3147d0");
+}
+
+#[test]
+fn test_divmod() {
```

Comment:
technically this is testing `div`, not `divmod`

Reply:
There's a macro named '/' which I added when there was controversy about this.  / was supposed to be deprecated.

Reply:
ah, right